### ISSUE TITLE
[1.2.0] Installation of templates changes files from GitHub sources

### ISSUE THREAD

**Issue by netniV** [Cacti/cacti#1719]
```
modified:   resource/snmp_queries/interface.xml
modified:   scripts/ss_host_cpu.php
modified:   scripts/ss_host_disk.php
```
The above files are changed on installation of template packages when compared to the GitHub sources.  These should be synchronised.

**Comment by cigamit** [comment]
Can you show a diff?

**Comment by netniV** [comment]
Difference appears to be tabs/spaces and the PHP_EOL vs \n

**Comment by cigamit** [comment]
Yea, we just need to repackage.

**Comment by cigamit** [comment]
Marking resolved for the 1.2 release.
